st2common/tests/integration/test_rabbitmq_ssl_listener.py

```diff
@@ -32,15 +32,16 @@
 ]
 
 CERTS_FIXTURES_PATH = os.path.join(get_fixtures_base_path(), 'ssl_certs/')
-ON_TRAVIS = (os.environ.get('TRAVIS', 'false').lower() == 'true')
+ST2_CI = (os.environ.get('ST2_CI', 'false').lower() == 'true')
 
 NON_SSL_LISTENER_PORT = 5672
 SSL_LISTENER_PORT = 5671
 
 
-# NOTE: We only run those tests on Travis because at the moment, local vagrant dev VM doesn't
-# expose RabbitMQ SSL listener by default
-@unittest2.skipIf(not ON_TRAVIS, 'Skipping tests because not running on Travis')
+# NOTE: We only run those tests on the CI provider because at the moment, local
+#       vagrant dev VM doesn't expose RabbitMQ SSL listener by default
+@unittest2.skipIf(not ST2_CI,
+                  'Skipping tests because ST2_CI environment variable is not set to "true"')
 class RabbitMQTLSListenerTestCase(unittest2.TestCase):
```

Comment:
This part is not skipped in TravisCI for some reason, according to
https://travis-ci.org/github/StackStorm/st2/builds/749891682#L9616-L9839
and so Travis Integration tests are failing now.

Reply:
I think these tests are intended to run on Travis CI. I think the Travis tests are failing because the RabbitMQ config file was changed.